src/main/java/com/synopsys/integration/jenkins/JenkinsProxyHelper.java

```diff
@@ -74,8 +73,8 @@ public JenkinsProxyHelper() {
         this.isBlank = true;
     }
 
-    public static JenkinsProxyHelper fromJenkins(Jenkins jenkins) {
-        ProxyConfiguration proxyConfig = Optional.ofNullable(jenkins)
+    public static JenkinsProxyHelper fromJenkins(JenkinsWrapper jenkinsWrapper) {
+        ProxyConfiguration proxyConfig = Optional.ofNullable(jenkinsWrapper.getJenkins()).orElse(null)
```

Comment:
This doesn't make sense since the `jenkinsWrapper.getJenkins()` returns an optional. So you don't need to wrap it in another Optional and you don't need the extra `.orElse(null)` the one at the end is fine.